vcd/resource_vcd_catalog_test.go

```diff
@@ -68,12 +86,12 @@ func testAccCheckVcdCatalogExists(name string, catalog *govcd.Catalog) resource.
 			return fmt.Errorf(errorRetrievingOrg, testConfig.VCD.Org+" and error: "+err.Error())
 		}
 
-		newCatalog, err := adminOrg.FindCatalog(rs.Primary.ID)
+		newCatalog, err := adminOrg.GetCatalogByNameOrId(rs.Primary.ID, false)
 		if err != nil {
-			return fmt.Errorf("catalog %s does not exist (%#v)", rs.Primary.ID, newCatalog)
+			return fmt.Errorf("catalog %s does not exist (%s)", rs.Primary.ID, err)
 		}
 
-		catalog = &newCatalog
+		catalog = newCatalog
```

Comment:
We don't use it. Do we?

Reply:
This is a copy-and-paste leftover from previous tests, where this pointer was used to help in some comparisons. In this test, it is not used, and it should not be needed if the resource `read` function does what it should. I will remove it.